### PR TITLE
Added Json schema for labelGrid

### PR DESCRIPTION
**Pull request by ivarne**
https://github.com/Altinn/altinn-studio/pull/7027

### CODE REVIEW

File: schemas/json/layout/layout.schema.v1.json [hunk 1]
```diff
@@ -194,6 +194,12 @@
     "gridSettings": {
         "allOf": [{ "$ref": "#/definitions/gridProps"}],
         "properties": {
+              "labelGrid": {
+                "title": "labelGrid",
+                "description": "Optional grid for label component. Used with innerGrid to make labels show on the side.",
```
Comment (lorang92) [inline schemas/json/layout/layout.schema.v1.json:199]:
```suggestion
                "description": "Optional grid for the component label. Used in combination with innerGrid to align labels on the side.",
```

**Review by lorang92 — CHANGES_REQUESTED** [review]

**Review by lorang92 — APPROVED** [review]
LGTM!

File: schemas/json/layout/layout.schema.v1.json [hunk 2]
```diff
@@ -194,6 +194,12 @@
     "gridSettings": {
         "allOf": [{ "$ref": "#/definitions/gridProps"}],
         "properties": {
+              "labelGrid": {
```
Comment (altinnadmin) [inline schemas/json/layout/layout.schema.v1.json:197]: Too indented? :)

**Review by altinnadmin — APPROVED** [review]
LGTM. One super-minor comment

File: schemas/json/layout/layout.schema.v1.json (hunk 2 above)
Comment (ivarne) [reply schemas/json/layout/layout.schema.v1.json:197]: The whole file is a mess, switching between 2 and 4 spaces indentation. Not an excuse for adding more inconcistency (6 spaces), but annoying when an autoformatter makes the file way less readable by expanding lots of arrays I'd rather keep on a single line.

Comment (altinnadmin) [reply schemas/json/layout/layout.schema.v1.json]: Heh, this PR just grew a little bit larger... :)
But cleanup up is good, and this cleanup looks good to me. But could you have a quick look aswell @lorang92 before I merge?

Comment (lorang92) [reply schemas/json/layout/layout.schema.v1.json]: @altinnadmin everything looks good to me! Great job @ivarne

**Pull request merged**